INSTALL.md

```diff
@@ -15,50 +32,51 @@ Update WSL's software before continuing. If you chose Debian, Ubuntu, or another
 apt-get update && apt-get upgrade
 ```
 
-WSL has its own file system that's not accessible from Windows, but Windows files *are* accessible from WSL. So you're going to want to install pokered within Windows. You'll have to change the **current working directory** every time you open WSL.
+WSL has its own file system that's not accessible from Windows, but Windows files *are* accessible from WSL. So you're going to want to install **pokered** within Windows. You'll have to change the **current working directory** every time you open WSL.
```

Comment:
```suggestion
WSL has its own file system that is accessible by navigating to `\\wsl$\<Distro Name>\` using a Windows Explorer window. Windows files are accessible from WSL by navigating to `/mnt/c/` (equivalent to Windows `C:\`) from within the WSL environment. Due to WSL's poor Windows file access times, especially for WSLv2, it is currently recommended that you download the **pokered** repo to and build from the WSL file system.

(Replace <Distro Name> with the name of the distro you installed into WSL. By default this is Ubuntu. For example: `\\wsl$\Ubuntu\`)
```

Reply:
````suggestion
WSL has its own file system that's stored within Windows. You can use Windows Explorer to access the WSL file system by navigating to `\\wsl$\<Distro Name>\`. (Replace <Distro Name> in the example path with the name of the distro you installed into WSL. By default this is Ubuntu. For example: `\\wsl$\Ubuntu\`.)

You can also use WSL to access the Windows file system by navigating to `/mnt/c/` (equivalent to Windows `C:\`) from within the WSL environment. WSL - especially WSLv2 - is slow to access Windows files, so it's recommended that you store **pokered** in WSL's file system and build it from there.

Every time you open WSL, you will have to change the **current working directory** to navigate to wherever it is you have stored **pokered**. For instance, if you want to store **pokered** in your WSL's home directory, enter this command:

```bash
cd /home/<user>/
```

(Replace `<user>` in the example path with your WSL username.)
````
Not sure if the bit about changing working directory is accurate. If you store pokered in `/home/<user>/` do you have to cd every time or is that only if you store somewhere else?

Reply:
WSL typically launches with the default current directory of your home directory `~` or the equivalent `/home/<user>`. Typically you would want to install pokered in the `/home/<user>/pokered`. So in that instance, yes you would need to cd into `pokered/` after launching WSL.

Reply:
OK, thank you for clarifying. How do you feel about this text?
````suggestion
WSL has its own file system that's stored within Windows. You can use Windows Explorer to access the WSL file system by navigating to `\\wsl$\<Distro Name>\`. (Replace <Distro Name> in the example path with the name of the distro you installed into WSL. By default this is Ubuntu. For example: `\\wsl$\Ubuntu\`.)

You can also use WSL to access the Windows file system by navigating to `/mnt/c/` (equivalent to Windows `C:\`) from within the WSL environment. WSL - especially WSLv2 - is slow to access Windows files, so it's recommended that you store **pokered** in WSL's file system and build it from there.

The default **current working directory** in WSL is your home directory, `/home/<user>/`. (Replace `<user>` in the example path with your WSL username.) If you store **pokered** in a different directory, you will have to change the **current working directory** every time you open WSL.  For instance, if you want to store **pokered** in `/home/example/` use this command:

```bash
cd /home/example/
```
````

Reply:
How do you feel about the suggested text?

Reply:
The example should be for storing inside your home user directory. `/home/<user>/`. 
That way when type type `git clone <pokered-repo-link>` the creates a folder for pokered. `/home/<user>/pokred/`

So rewriting the last sentence should say somethin like. ``For instance, if you want to store **pokered** in your home user directory `/home/<user>/` use this command:``

` ```bash`
` cd /home/<user>/ `
` ``` `